portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase

```diff
@@ -105,6 +105,16 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-96545. The edited default values are kept when cancel the Reset Values action in imported structure."
+	@ignore = "true"
+	@priority = "2"
+	test CancelResetImportedStructureDefaultValues {
+		property portal.upstream = "false";
+
+		// TODO LPS-123456 CancelResetImportedStructureDefaultValues pending implementation
```

Comment:
Sorry I did a mistake in my example PR, this LPS should be the one related to the story we are automating. LPS-96545 in this case.

Same for all // TODO comments

Reply:
Never mind. I fixed it locally :)